doc/classes/Joint2D.xml

```diff
@@ -74,12 +74,16 @@
 	</methods>
 	<members>
 		<member name="bias" type="float" setter="set_bias" getter="get_bias">
+			The force with which nodes A and B stay in the same relative position to the joint over time.
```

Comment:
This is still a really difficult sentence to parse.  I don't really have any idea what's going on with this property.

Reply:
ok, will explain more :)